docker/dockerfiles/Dockerfile

```diff
@@ -132,22 +133,29 @@ WORKDIR /tmp
 # en_US.UTF-8 locale not available by default
 RUN locale-gen en_US.UTF-8
 
-# install mysql
+# Install MySQL.
+#
+# We explicitly target MySQL 5.7, and because of that we also explicitly target
+# Ubuntu Bionic (18.x).
+#
+# We also explicitly install both mysql-client and mysql-server, so that we can
+# also explicitly define the versions of both; normally only the latter is
+# required since it depends on the former.
+#
+# Once we update to MySQL 8+, this can be simplified.
```

Comment:
I appreciate that this comment will be discoverable by anyone searching for "mysql 5.7" or "ubuntu". Not sure if you've been intentional about that in all these PR's, but if so, thanks!

Reply:
Yeah! I've learned through doing all these upgrades that explicit library names and version numbers in comments are super valuable, particularly in future-facing comments like this one.